sql/scanner/token.go

```diff
@@ -221,14 +236,25 @@ var tokens = [...]string{
 	WHERE:       "WHERE",
 	WRITE:       "WRITE",
 
-	TYPEARRAY:    "ARRAY",
-	TYPEBLOB:     "BLOB",
-	TYPEBOOL:     "BOOL",
-	TYPEBYTES:    "BYTES",
-	TYPEDOCUMENT: "DOCUMENT",
-	TYPEDOUBLE:   "DOUBLE",
-	TYPEINTEGER:  "INTEGER",
-	TYPETEXT:     "TEXT",
+	TYPEARRAY:       "ARRAY",
```

Comment:
We should remove the `TYPE` prefix for all the types and order them in alphabetical order like the rest of the keywords

Reply:
I didn't dare...